### PR TITLE
Add user + dev meeting archive

### PR DESCRIPTION
**Pull request by ljflores**

### CODE REVIEW

**Comment by ljflores** [comment]
https://wip-user-dev-files.ceph.io/

**Comment by ljflores** [comment]
https://wip-user-dev-files.ceph.io/en/community/meetups/
https://wip-user-dev-files.ceph.io/en/community/meetups/user-dev-archive/

File: src/en/community/meetups/index.html [hunk 1]
```diff
@@ -241,7 +241,9 @@ <h3 class="h3">Ceph User + Dev Meetup</h3>
       <a class="a block mb-4" href="https://meet.jit.si/ceph-user-dev-monthly" target="_blank" rel="noreferrer noopener"
         >Join the meeting</a
       >
-      <a class="a" href="https://pad.ceph.com/p/ceph-user-dev-monthly-minutes" target="_blank" rel="noreferrer noopener">Etherpad</a>
+      <a class="a" href="https://pad.ceph.com/p/ceph-user-dev-monthly-minutes" target="_blank" rel="noreferrer noopener">Etherpad</a
+      >
+      <a class="a" href="/{{ locale }}/community/meetups/user-dev-archive" target="_blank" rel="noreferrer noopener">Archive</a>
```
Comment (neha-ojha) [inline src/en/community/meetups/index.html:246]: It is showing a 404 error in the current format. https://wip-user-dev-files.ceph.io/en/assets/pdfs/user_dev_meeting_2023_09_21_cory_snyder.pdf

Comment (ljflores) [reply src/en/community/meetups/index.html:246]: Will fix

Comment (ljflores) [reply src/en/community/meetups/index.html:246]: It's still 404: https://ceph.io/assets/pdfs/user_dev_meeting_2023_09_21_cory_snyder.pdf

But this is the correct convention, to hardcode https://ceph.io. Once merged, it should point correctly to the file.

**Review by neha-ojha — APPROVED** [review]

**Comment by ljflores** [comment]
Thx Neha! Merging!

**Pull request merged**